src/components/Banner/MigrationV2.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { Text, Flex, Box, ErrorIcon, Button } from '@pancakeswap-libs/uikit'
+import Banner from './Banner'
+
+const MigrationV2 = () => {
+  return (
+    <Banner id="v2-migration">
+      <Flex mb="16px">
+        <ErrorIcon color="white" width="32px" mr="16px" />
+        <Text color="white" fontSize="24px" bold>
+          ACTION REQUIRED for all LP token holders
+        </Text>
+      </Flex>
+      <Box ml="48px">
+        <Text color="warning" mb="16px" bold>
+          You must complete migration if you want to keep earning from any LP tokens that aren’t staked in PancakeSwap
```

Comment:
All LGTM. I think this text is a bit misleading? Should "aren't" become **are**?

> You must complete migration if you want to keep earning from any LP tokens that **are** staked in PancakeSwap

Reply:
I just copy Bug's text

Reply:
Confirmed with Chungus and Bugs, the text should be corrected to **are**